### PR TITLE
fix: Correct the afc path and handle the case of documents container type

### PR DESCRIPTION
**Pull request by umutuzgur**

### CODE REVIEW

**Comment by umutuzgur** [comment]
Resolve this issue https://github.com/appium/appium/issues/12983

File: lib/commands/file-movement.js [hunk 1]
```diff
@@ -54,13 +55,17 @@ async function createAfcClient (udid, bundleId, containerType) {
...
-  if (containerType === IFUSE_CONTAINER_DOCUMENTS) {
+  if (isDocuments(containerType)) {
     return await service.vendDocuments(bundleId);
   } else {
     return await service.vendContainer(bundleId);
   }
 }
 
+function isDocuments (containerType) {
+  return containerType.toLowerCase() === IFUSE_CONTAINER_DOCUMENTS;
```
Comment (KazuCocoa) [inline lib/commands/file-movement.js]: `_.toLower ` since `containerType` looks it can be `undefiend`

**Comment by umutuzgur** [comment]
@KazuCocoa It is fixed now. `path.resolve` works different than java apparently

**Comment by KJAbhilash** [comment]
Getting same error when trying to push files of format png and txt.
Using 1.15.1 version of appium desktop.

**Pull request merged**